src/Web/Startup.cs

```diff
@@ -19,26 +20,31 @@ public Startup(IConfiguration configuration)
     private IConfiguration Configuration { get; }
 
     // This method gets called by the runtime. Use this method to add services to the container
-    public void ConfigureServices(IServiceCollection services)
+    public async void ConfigureServices(IServiceCollection services)
     {
         services.AddControllers();
-        services.AddSingleton<IBookService, BookService>();
-        services.AddSingleton<IBookDynamoDbStorageService, BookDynamoDbStorageService>();
         var awsOptions = new AWSOptions()
         {
             Region = RegionEndpoint.EUWest2
         };
 
         var env = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
-        if (env == "LOCAL")
-        {
-            new LocalDynamoDbSetup().SetupDynamoDb();
-            awsOptions.Credentials = new BasicAWSCredentials("x", "x");
-            awsOptions.DefaultClientConfig.ServiceURL = "http://localhost:8000";
-        }
+        // if (env == "LOCAL")
+        // {
+        //     new LocalDynamoDbSetup().SetupDynamoDb();
+        //     awsOptions.Credentials = new BasicAWSCredentials("x", "x");
+        //     awsOptions.DefaultClientConfig.ServiceURL = "http://localhost:8000";
+        // }
         services.AddDefaultAWSOptions(awsOptions);
-        
         services.AddAWSService<IAmazonDynamoDB>(awsOptions);
+        services.AddAWSService<IAmazonS3>();
+        
+        services.AddSingleton<IBookDynamoDbStorageService, BookDynamoDbStorageService>();
+        services.AddSingleton<IMemeDynamoDbStorageService, MemeDynamoDbStorageService>();
+        services.AddSingleton<IBookService, BookService>();
```

Comment:
Was just wondering why these are added as singletons instead of scoped?

Reply:
I think we only need one instance in the lifetime of the app as none of the properties are ever expected to change. The services have the dynamodb client, s3 client and the bucket name all which are static and shouldn't ever change to warrant a scoped service